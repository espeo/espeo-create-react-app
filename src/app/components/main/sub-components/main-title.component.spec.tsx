import { MainTitle } from './main-title.component';

import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('Main Title Component', () => {
  test('should be defined', () => {
    const component = renderer.create(<MainTitle text='title' />);
    expect(component).toBeDefined();
  });

  test('should match snapshot', () => {
    const component = renderer.create(<MainTitle text='title' />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should render string from title attribute', () => {
    const text = shallow(<MainTitle text='title' />).find('h1').text();
    expect(text).toEqual('title');
  });
});
