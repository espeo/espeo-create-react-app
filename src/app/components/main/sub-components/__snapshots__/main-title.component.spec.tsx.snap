// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Title Component should match snapshot 1`] = `
<h1>
  title
</h1>
`;
